README.md

```diff
@@ -1,2 +1,24 @@
 # -Hello-Microverse-project
 "Hello Microverse" project first project Day2
+
+
+Hello-Microverse-project
+Description the project.
+
+Built With
+Major languages HTML & CSS
+Frameworks
+Technologies used
+Live Demo (if available)
+
+
+
+Authors
+👤 Paul Malongo
+
+GitHub: @canon20paul
+Twitter: @paulmalongo
+LinkedIn: Paulmalongo
+
+ License
+This project is MIT licensed.
```

Comment:
Your readme file does not match the requirements of the project. Refer to [this](https://github.com/microverseinc/readme-template/blob/master/README.md) readme file on how to customize your readme file so it looks professional.